### PR TITLE
Support user specified basePath. fixes #153

### LINKED ISSUE #153: Support user-specified basePath
Allowing the developer to specify the installation directory and returning the `fsPaths` mapping allows for greater flexibility and control over where the `jar` files are installed to. This also allows developers easy programmatic access to the jars, which may be required in some applications (eg: nightwatch).

### PR DESCRIPTION
**Pull request by knksmith57**

### CODE REVIEW

File: lib/install.js [hunk 1]
```diff
@@ -75,7 +76,9 @@ function install(opts, cb) {
     tasks.push(chmodChromeDr.bind(null, fsPaths.chrome.installPath));
   }
 
-  async.series(tasks, cb);
+  async.series(tasks, function(err) {
+    cb(err, fsPaths);
```
Comment (vvo) [inline lib/install.js]: where is this used?

Comment (vvo) [inline lib/install.js]: Ok got it

**Comment by vvo** [comment]
Can you add some documentation?

**Comment by vvo** [comment]
Also,...

AWESOME!

**Comment by knksmith57** [comment]
@vvo added notes about `opts.basePath` to the `selenium.install` and `selenium.start` sections of the README. How's that look (or did you have something else in mind for the docs)?

**Comment by vvo** [comment]
Perfect! Merging

**Comment by vvo** [comment]
released as 4.8.0 with other fixes

**Pull request merged**